domain/cloud/state/state.go

```diff
@@ -340,9 +340,11 @@ SELECT cloud.uuid, cloud.name, cloud_type_id,
        auth_type.type, cloud_type.type,
        COALESCE((SELECT true
```

Comment:
Is SQLite smart enough not to run this for every row? It's inefficient if not.

In any case, it's clearer to put this statement in a `with` block up front, and use an outer join.